scripts/vote_2025_07_16.py

```diff
@@ -0,0 +1,241 @@
+14. Grant MODULE_MANAGER_ROLE on CSModule 0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F to Aragon Agent 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c
+15. Reduce keyRemovalCharge from 0.02 to 0 ETH on CSModule 0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F
+16. Revoke MODULE_MANAGER_ROLE on CSModule 0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F from Aragon Agent 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c
+
+IV. CS Verifier rotation
+17. Revoke VERIFIER_ROLE role on CSModule 0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F from old CS Verifier 0x0c345dFa318f9F4977cdd4f33d80F9D0ffA38e8B (addresses to be verified and checked)
+18. Grant VERIFIER_ROLE role on CSModule 0xdA7dE2ECdDfccC6c3AF10108Db212ACBBf9EA83F to new CS Verifier 0xeC6Cc185f671F627fb9b6f06C8772755F587b05d (addresses to be verified and checked)
+"""
+
+import time
+import importlib
+import utils.voting
+importlib.reload(utils.voting)
+
+from typing import Dict
+
+from brownie import interface
+from brownie.network.transaction import TransactionReceipt
+
+from utils.mainnet_fork import pass_and_exec_dao_vote
+from utils.agent import agent_forward
+from utils.dual_governance import submit_proposals
+from utils.voting import create_vote, bake_vote_items, confirm_vote_script
+from utils.easy_track import remove_evmscript_factory
+from utils.ipfs import calculate_vote_ipfs_description, upload_vote_ipfs_description
+from utils.config import (
+    contracts,
+    get_deployer_account,
+    get_is_live,
+    get_priority_fee,
+)
+
+from utils.permissions import (
+    encode_oz_revoke_role,
+    encode_oz_grant_role
+)
+
+HASH_CONSENSUS_FOR_ACCOUNTING_ORACLE_QUORUM = 5
+HASH_CONSENSUS_FOR_VALIDATORS_EXIT_BUS_ORACLE_QUORUM = 5
+HASH_CONSENSUS_FOR_CS_FEE_ORACLE_QUORUM = 5
+
+KYBER_ORACLE_MEMBER = "0xA7410857ABbf75043d61ea54e07D57A6EB6EF186"
+CALIBER_ORACLE_MEMBER = "0x4118DAD7f348A4063bD15786c299De2f3B1333F3"
+
+NEW_KEY_REMOVAL_CHARGE = 0
+
+CS_VERIFIER_ADDRESS_OLD = "0x0c345dFa318f9F4977cdd4f33d80F9D0ffA38e8B"
+CS_VERIFIER_ADDRESS_NEW = "0xeC6Cc185f671F627fb9b6f06C8772755F587b05d"
+
+STAKING_MODULE_ID = 3
+STAKE_SHARE_LIMIT_NEW = 300
+PRIORITY_EXIT_SHARE_THRESHOLD_NEW = 375
+
+# TODO: To be defined
+IPFS_DESCRIPTION = ""
+
+
+def start_vote(tx_params: Dict[str, str], silent: bool) -> bool | list[int | TransactionReceipt | None]:
+    """Prepare and run voting"""
+
+    # Contracts definition
+    hash_consensus_for_accounting_oracle: interface.LidoOracle = contracts.hash_consensus_for_accounting_oracle
+    hash_consensus_for_validators_exit_bus_oracle: interface.LidoOracle = contracts.hash_consensus_for_validators_exit_bus_oracle
+    csm_hash_consensus: interface.CSHashConsensus = contracts.csm_hash_consensus
+    csm_module = contracts.staking_router.getStakingModule(STAKING_MODULE_ID)
+
+    combined_call_script = [
```

Comment:
items executed by Voting contract:
```suggestion
    voting_call_script = [
```